src/ralph/admin/autocomplete_urls.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from django.conf.urls import url
+
+from ralph.admin.autocomplete import AutocompleteList
+
+urlpatterns = [
+    url(
+        r'^autocomplete/(?P<app>\w+)/(?P<model>\w+)/(?P<field>\w+)',
```

Comment:
wouldn't it be better to use following url:
```
^(?P<app>\w+)/(?P<model>\w+)/(?P<field>\w+)/autocomplete$
```
Then `^(?P<app>\w+)/(?P<model>\w+)/(?P<field>\w+)` could be used to display list of available items after you click on magnifier icon next to autocomplete.